Lib/test/support/__init__.py

```diff
@@ -2907,10 +2907,11 @@ def in_systemd_nspawn_sync_suppressed() -> bool:
     # trigger EINVAL.  Otherwise, ENOENT will be given instead.
     import errno
     try:
-        with os.open(__file__, os.O_RDONLY | os.O_SYNC):
-            pass
+        file_no = os.open(__file__, os.O_RDONLY | os.O_SYNC)
```

Comment:
Can you rename the variable to "fd"? It's more common to refer to file descriptors as "fd".

Reply:
Done.